data/com.endlessm.Clippy.Extension.appdata.xml

```diff
@@ -0,0 +1,12 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<component type="runtime">
+  <id>com.endlessm.Clippy.Extension</id>
+  <extends>com.endlessm.HackComponents</extends>
+  <metadata_license>LicenseRef-proprietary</metadata_license>
+  <project_license>LicenseRef-proprietary</project_license>
+  <name>Clippy</name>
+  <summary>Clips things together</summary>
+  <vetos>
+    <veto>NoDisplay=true</veto>
+  </vetos>
```

Comment:
I thought that this was put there automatically by the build process? Do we need it here?

Reply:
It does, but only when there's a desktop with the `NoDisplay` entry as well AFAIK.

Reply:
This means that Avatars should have it too. I will add it.

Reply:
It does not look like this should be required after all. None of the `type=addon` bundles on flathub, for instance have it.

Reply:
Right. I will remove it.